os/include/tinyara/spi/spi.h

```diff
@@ -389,6 +392,22 @@ enum spi_mode_e {
 	SPIDEV_MODE3				/* CPOL=1 CHPHA=1 */
 };
 
+/*
+ * Structure for a communication between kernel and user spac
+ */
+struct spi_io_config {
+	unsigned int bpw;
```

Comment:
We using macro bits, mode, frequency, status is needed, also using macro based on this.
So status should be added more and we'd better rename bpw to bits, cs to port.